VerseReportingProcessor/VerseCounterService.cs

```diff
@@ -52,7 +52,7 @@ public async Task StartAsync(CancellationToken cancellationToken)
 	    {
 		    var body = args.Message.Body.ToString();
 			var input = JsonSerializer.Deserialize<VerseCountingResult>(body);
-			if (input == null)
+			if (input?.LanguageCode == null || input.RepoId == 0 || input.RepoId == null || input.Repo == null)
```

Comment:
Should the other references to `input` also use the elvis operator?  e.g. `input?.RepoId`?

Reply:
If input is null at all the first one will catch it so no need to check the others